manticore/core/smtlib/solver.py

```diff
@@ -126,9 +126,14 @@ def __init__(self):
         self.support_reset = True
         logger.debug('Z3 version: %s', self.version)
 
-        if self.version >= Version(4, 4, 1):
+        if self.version >= Version(4, 5, 0):
+            self.support_maximize = False
+            self.support_minimize = False
```

Comment:
was `maximize`/`minimize` support removed in 4.5.0?

Reply:
No, they just changed the format. Stuff that was printed out by default is not  any more. And now there is a special separated command to ask for "objectives"

Reply:
Should we add an issue for adding support for the new z3 ? 
(Our current parsing wont work)